lib/std/Build/Fuzz.zig

```diff
@@ -155,7 +155,7 @@ fn fuzzWorkerRun(
             const stderr = std.io.getStdErr();
             std.debug.lockStdErr();
             defer std.debug.unlockStdErr();
-            build_runner.printErrorMessages(gpa, &run.step, ttyconf, stderr, false) catch {};
```

Comment:
Note that this will show error traces for the fuzzer, I don't know if it is relevant though, I'm not familiar enough with the fuzzing infra.